hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupSystemTable.java

```diff
@@ -1902,4 +1905,19 @@ private static void ensureTableEnabled(Admin admin, TableName tableName) throws
       }
     }
   }
+
+  /**
+   * Executes the given operations in partitioned batches of size {@link #BATCH_SIZE}
+   */
+  private static void executePartitionedBatches(Table table, List<? extends Row> operations)
+    throws IOException {
+    List<? extends List<? extends Row>> operationBatches = Lists.partition(operations, BATCH_SIZE);
+    for (List<? extends Row> batch : operationBatches) {
+      try {
+        table.batch(batch, new Object[batch.size()]);
```

Comment:
Look at the implementation of `HTable`. `Table#put(List)` does exactly this already. If you want to change the edit-shipping semantics, this won't do it.